bin/render-activity-stream-html.js

```diff
@@ -111,11 +109,14 @@ function main() {
   const {html, state} = prerender(options.locale);
   options.direction = state.App.textDirection;
 
-  fs.writeFileSync(HTML_FILE_PATH, templateHTML(options));
-  fs.writeFileSync(PRERENDERED_HTML_FILE_PATH, templateHTML(options, html));
-  fs.writeFileSync(INITIAL_STATE_JS_FILE_PATH, templateJs(state));
+  fs.writeFileSync(path.join(BASE_FILE_PATH, "activity-stream.html"), templateHTML(options));
+  fs.writeFileSync(path.join(BASE_FILE_PATH, "activity-stream-debug.html"), templateHTML(options, null, true));
```

Comment:
Looks like you want options on your options! ;) And that I believe passing a bool argument is generally bad API design. But ah well, it's a `bin` script.

Reply:
Generally I definitely agree... I could object assign again I suppose